recipe/meta.yaml

```diff
@@ -23,35 +23,35 @@ source:
     sha256: 151ce8f50b9e65082efc1edaf0aa01b5a96650f7226ae18a3b47af286dfd62fb
 
 build:
-  number: {{ build }}
-  skip: true  # [py < 39 or osx or ppc64le]
+  number: {{ number }}
+  skip: true  # [(py < 39) or osx or ppc64le]
+
+requirements:
+  host:
+    - python
```

Comment:
This is the key to 
1. suppress skipped Python pipelines
2. stabilize rerendering (so that it wouldn't take 2 rerenders when the infra is changed)

Reply:
See also https://github.com/conda-forge/cupy-feedstock/pull/241#issuecomment-1900744234